tests/41end-to-end-keys/08-cross-signing.pl

```diff
@@ -682,11 +682,21 @@
       })->then( sub {
          sync_until_user_in_device_list( $user1, $user2 );
```

Comment:
hrm. I believe the intention of this line is to wait for the device info to become available. Do you know why it's insufficient? (And if there's a good reason, we may as well remove it.)

Reply:
I think
- this is explained by https://github.com/matrix-org/sytest/pull/1169#discussion_r754194531.
- this `then` call is therefore redundant.